operator/pkg/securedcluster/extensions/reconcile_scanner_v4_db_password.go

```diff
@@ -0,0 +1,45 @@
+package extensions
+
+import (
+	"context"
+
+	"github.com/go-logr/logr"
+	"github.com/operator-framework/helm-operator-plugins/pkg/extensions"
+	platform "github.com/stackrox/rox/operator/apis/platform/v1alpha1"
+	commonExtensions "github.com/stackrox/rox/operator/pkg/common/extensions"
+	"github.com/stackrox/rox/operator/pkg/securedcluster/scanner"
+	ctrlClient "sigs.k8s.io/controller-runtime/pkg/client"
+)
+
+var _ commonExtensions.ScannerBearingCustomResource = (*securedClusterWithScannerBearer)(nil)
```

Comment:
just noticed this should be `securedClusterWithScannerV4Bearer`. Sorry this whole V4 thing is being a huge pain

Reply:
Done.